pkiintergration-fe/src/modules/pki-managed-certificates/create-certificate/create-certificate.component.html

```diff
@@ -28,18 +28,6 @@ <h4 class="text-uppercase" style="margin: 0; letter-spacing: 0.15em; color: whit
     <div class="modal-footer">
       <button class="btn btn-default" translate="" (click)="bsModalRef.hide()">Cancel</button>
       <button type="submit" class="btn btn-primary" (click)="bsModalRef.hide()">Generate</button>
```

Comment:
question: why do both buttons trigger the same action/function - where is the difference?